app/src/main/java/space/taran/arknavigator/mvp/model/repo/scores/PlainScoreStorage.kt

```diff
@@ -103,8 +104,10 @@ class PlainScoreStorage(
             verifyVersion(version)
             val result = lines.map {
                 val parts = it.split(KEY_VALUE_SEPARATOR)
-                val id = parts[0].toLong()
-                val score = parts[1].toInt()
+                val id = ResourceId.create(
+                    parts[0].toLong(), parts[1].toLong()
+                )
+                val score = parts[2].toInt()
```

Comment:
`create` method is needed only for JNI, you can create an object in the usual way.

We need to decide how to store `ResourceId` in text and encapsulate serialization logic inside `ResourceId`